whole-repo-tests/test_pyright.py

```diff
@@ -173,6 +173,26 @@ def test_numpy_arrays_strategy(tmp_path: Path):
     assert errors == []
 
 
+def test_pandas_column(tmp_path: Path):
+    file = tmp_path / "test.py"
+    file.write_text(
+        textwrap.dedent(
+            """
+            from hypothesis.extra.pandas import column
+            from hypothesis.strategies import floats
+
+            x = column(name="test", elements=floats(), fill=None, unique=True)
+            y = column(name="test", dtype=float, fill=None, unique=True)
```

Comment:
I'd love to have a test that asserts we see the expected type for the column, and to include a similar test in `test_mypy.py`.